service_tools/package.xml

```diff
@@ -0,0 +1,18 @@
+<package format="2">
+  <name>service_tools</name>
+  <version>0.0.0</version>
```

Comment:
could you set the version to ```0.6.4``` - otherwise we'll have some trouble on the next release